jwst/ramp_fitting/utils.py

```diff
@@ -543,7 +543,7 @@ def calc_slope_vars(rn_sect, gain_sect, gdq_sect, group_time, max_seg):
         del wh_good
 
         # Locate any CRs that appear before the first SAT group...
-        wh_cr = np.where( gdq_2d_nan[i_read, :] == dqflags.group['JUMP_DET'])
+        wh_cr = np.where( gdq_2d_nan[i_read, :] & dqflags.group['JUMP_DET'] > 0 )
```

Comment:
Unfortunately `gdq_2d_nan` is a float here, so it won't work.  Not sure why it needs to be a float.

https://github.com/spacetelescope/jwst/blob/2b54a372d588091f2faa9a9e212f59ff132c3174/jwst/ramp_fitting/utils.py#L519

Reply:
Whoops!  Guess it's a float to support nan values?  Anyway, I'll close this, seems like my "fix" would do more harm than good.

Reply:
Aaaaaand it's back!